app/Models/Webhook.php

```diff
@@ -6,5 +6,11 @@
 
 class Webhook extends Model
 {
-    //
+    public function user() {
+        return $this->belongsTo(User::class);
+    }
+
+    public function bot() {
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Squiz.Functions.MultiLineFunctionDeclaration.BraceOnSameLine</br>**Severity**: error</br>**File**: app/Models/Webhook.php</br>**Line**: 13</br>Opening brace should be on a new line